src/type/array_type.cpp

```diff
@@ -12,13 +12,15 @@
 
 #include "type/value.h"
 #include "type/array_type.h"
+#include <sstream>
```

Comment:
Is this needed?

Reply:
Yes, it's needed for ArrayType::ToString